reactor-netty-http/build.gradle

```diff
@@ -151,6 +151,15 @@ dependencies {
 	testRuntimeOnly "org.slf4j:jcl-over-slf4j:$slf4jVersion"
 	testRuntimeOnly "ch.qos.logback:logback-classic:$logbackVersion"
 
+	// Needed for Brotli compression
+	testImplementation "com.aayushatharva.brotli4j:brotli4j:$brotli4jVersion"
+	if (osdetector.classifier == "linux-aarch_64" || osdetector.classifier == "osx-aarch_64") {
```

Comment:
What about Armv7?

Reply:
@hyperxpro This dependency is just for tests. What's the classifier for Armv7?

Reply:
`native-linux-armv7`

If it's just for test then it should be fine but I'd recommend adding just in case someone compiles and tests it on Armv7.

Reply:
let's skip it for the moment, we can add this later if needed